schema_utils.py

```diff
@@ -373,3 +375,108 @@ def is_valid_email(obj):
             bool. Whether the given object is a valid email.
         """
         return bool(re.search(r'^[\w\.\+\-]+\@[\w]+\.[a-z]{2,3}$', obj))
+
+
+class CustomizationArgsUtil(object):
```

Comment:
Hmm, if you're going this route (which I like!), maybe we should just introduce a separate module: `core.domains.customization_args_util`?

Reply:
Yes, that is definitely better. I have made this change, PTAL!